### PR TITLE
Emacs Cheat Sheet text wrapped in braces 

### LINKED ISSUE #1281: Emacs CheatSheets: wrap text snippets in angle brackets; convert to code block
The Emacs cheat sheet has a few "keys" which include a key, then a text entry, and then another key:

`M-x query-replace RET`

This should be converted to:

`[M-x] [<query-replace>] [RET]`

The result should look like this:
emacs_help_at_duckduckgo

------
IA Page:  http://duck.co/ia/view/cheat_sheets

### PR DESCRIPTION
**Pull request by Sloff**
Fixes #1281 

**Before**
before

**After**
after

### CODE REVIEW

**Comment by mintsoft** [comment]
@Sloff Whilst you're here, would you mind wrapping all keys in `[ ]` ?

**Comment by Sloff** [comment]
Hi @mintsoft, I'm on it!

**Comment by mintsoft** [comment]
Excellent thanks!

**Comment by Sloff** [comment]
@mintsoft, shoud I wrap the single keys as ```"key": "[ C ]"```? Because when I try ```"key": "[C]"``` the braces are shown in the cheat sheet

**Comment by Sloff** [comment]
@mintsoft, sorry for bothering you, I'm not completely sure what I should do. Should I add ```[ ]``` to all the keys? For example ```"key": "C"```, or should I change the ones like this: ```"key": "C-f"``` to ```"key": "[C] [f]"```?

**Comment by mintsoft** [comment]
@Sloff I think any keys should have `[ ]` around them, so `C-f` would become `[C]+[f]`

**Comment by mintsoft** [comment]
@Sloff There's actually docs on this (I always forget!): https://duck.co/duckduckhack/goodie_cheat_sheets#formatting-key-presses

**Comment by moollaza** [comment]
>or should I change the ones like this: "key": "C-f" to "key": "[C] [f]"?

@Sloff @mintsoft we agreed that the *nix style `C-x` is acceptable because it's common amongst lots of *nix programs and docs (e.g. Emacs) so it can remain in that style.

File: .travis.yml [hunk 1]
```diff
@@ -16,4 +16,4 @@ language: perl
 perl:
   - 5.16
 script:
-  - dzil smoke --release --author 2> /dev/null
+  - dzil smoke --release --author
```
Comment (moollaza) [inline .travis.yml]: @Sloff uh oh. It looks like you've accidentally pulled in some changes that aren't supposed to be here?

**Comment by Sloff** [comment]
@moollaza, sorry about that, still getting the hang of git and github, going to close this and open a new pull request

**Comment by mintsoft** [comment]
@moollaza Thanks for the clarification! I shall remember that for the future :+1:

**Pull request closed without merging**